guides/release/getting-started/core-concepts.md

```diff
@@ -74,10 +74,10 @@ In the following example, the [`didRender()`](https://emberjs.com/api/ember/rele
 ```javascript {data-filename=/app/components/foo-did-render-example.js}
 import Component from '@ember/component';
 
-export default Component.extend({
+export default class FooDidRenderExample extends Component {
   didRender() {
-    this._super(...arguments);
+    super(...arguments);
```

Comment:
I'm not that familiar with Class syntax, but should this be `super.didRender(...arguments)`? (I think I read that somewhere but I could be mistaken)

Reply:
But super is a function right, how can you call `super.didRender`, generally to call the base class function we will be using `super()` with whatever arguments from the child class

Reply:
Found something useful on MDN
Your syntax works on static functions i guess
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/super

But from the implementation of didRender, i don't think that is a static function
https://github.com/emberjs/ember.js/blob/30137796af42c63b28ead127cba0e43e45a773c1/packages/%40ember/-internals/glimmer/lib/component.ts#L798

Please let me know whether i am going in the right direction...

Reply:
This should definitely be 
```suggestion
    super.didRender(...arguments);
```
This does not only work on static methods. I think the MDN article is just noting that it also works on static methods. It's strange that they don't mention it works for regular class methods. It [definitely works](https://ember-twiddle.com/81eb88e1b46a337d9c9af46428b3467e?openFiles=components.child-component.js%2Ctemplates.components.child-component.hbs).